lib/irc.js

```diff
@@ -68,7 +68,8 @@ function Client(server, clientNick, opt) {
         encoding: null,
         millisecondsOfSilenceBeforePingSent: 15 * 1000,
         millisecondsBeforePingTimeout: 8 * 1000,
-        enableStrictParse: false
+        enableStrictParse: false,
+        extendedJoin: false
```

Comment:
If you could add a note to the API documentation about `extendedJoin`, that would be great.